examples/check.py

```diff
@@ -2,27 +2,31 @@
 
 from kessel.inventory import v1beta2
 from kessel.inventory.v1beta2 import (
```

Comment:
are these imports going to stay the same once we release it as `kessel-sdk` ?

Reply:
Great question, yes they are the same imports but looks like there was a small issue with setup for packaging. We were only taking the top level import and missing everything else